### ISSUE TITLE
Feature request - disabling sniff per file/line

### ISSUE THREAD

**Issue by polishdeveloper** [joakin/resource-modules#2]
Before we add this tool as a -2 check to CI we need a possibility to disable check per file (or line). There is a possibility where 
 - tool rejects valid code 
 - we're in experimenting/prototyping phase and there is no need to meet all standards as we just want to demo something
 - we start using a new tool/library that conflicts with resource-modules linter

**Comment by joakin** [comment]
I'm thinking something along the lines of:
```js
mw.msg('some-msg-not-defined-explicitly') // resource-modules-disable-line
```
Would be best initially. If we see a need to disable parsing in a block of code we could add `// resource-modules-enable` and `// resource-modules-disable`.

I see acorn can collect comments with `onComment` when calling `parse` so it should be doable somehow.

**Comment by joakin** [comment]
Resolved for now in 1.0.17 https://github.com/joakin/resource-modules/compare/v1.0.15...v1.0.17

See README.
